<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-EA4CD3Q6" failures="2" tests="2" name="Default test(failed)" time="0.072" errors="0" timestamp="2021-11-18T13:47:37 IST">
  <testcase classname="failed_test_cases2.Failed_test_cases_run2" name="mno" time="0.015">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at failed_test_cases2.Failed_test_cases_run2.mno(Failed_test_cases_run2.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- mno -->
  <testcase classname="failed_test_cases2.Failed_test_cases_run2" name="Abc" time="0.002">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at failed_test_cases2.Failed_test_cases_run2.Abc(Failed_test_cases_run2.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Abc -->
  <testcase classname="failed_test_cases2.Failed_test_cases_run2" name="pqr" time="0.0">
    <ignored/>
  </testcase> <!-- pqr -->
  <testcase classname="failed_test_cases2.Failed_test_cases_run2" name="def" time="0.0">
    <ignored/>
  </testcase> <!-- def -->
  <testcase classname="failed_test_cases2.Failed_test_cases_run2" name="xyz" time="0.0">
    <ignored/>
  </testcase> <!-- xyz -->
</testsuite> <!-- Default test(failed) -->
